src/qt/rpcconsole.cpp

```diff
@@ -884,9 +884,21 @@ void RPCConsole::setNumBlocks(int count, const QDateTime& blockDate, double nVer
     }
 }
 
-void RPCConsole::setMasternodeCount(const QString &strMasternodes)
+void RPCConsole::setMasternodeCount()
 {
-    ui->masternodeCount->setText(strMasternodes);
+    if (!clientModel) {
+        return;
+    }
+    CDeterministicMNList mnList;
+    clientModel->getMasternodeList(mnList);
+    // QString strMasternodeCount = tr("Total: %1 (PS compatible: %2 / Enabled: %3) (IPv4: %4, IPv6: %5, TOR: %6)").arg(QString::number((int)mnodeman.size()))
```

Comment:
maybe it's time to remove these commented out versions of the string? They always caused more confusion to me then they helped.